doc/changes/changes_1.1.2.md

```diff
@@ -0,0 +1,39 @@
+# error-code-crawler-maven-plugin 1.1.2, released 2022-??-??
```

Comment:
I want to implement more tickets, no release planned yet.

Reply:
noted